src/python-common/ceph/deployment_utils/ssh_orchestrator.py

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
I do not understand this very well. Is this an example/recipe?.... I think that the ssh orchestrator (and others) must be inside mgr.

Reply:
having this code outside of the mgr has some real benefits:

1. Code is much easier to test: If we don't need to start a whole cluster, we can just use tox to do some real testing. 
2. There is a possibility to attach a CLI to let admins run those commands on a host to help with debugging issues
3. There is a bootstrapping issue that could be solved by (2.)